### PR TITLE
Custom post path parameter

### LINKED ISSUE #319: Make POST path of Emitter configurable
The release 1.1.0 of the objective-C tracker has a new postPath parameter that makes Snowplow emitter configurable, following an approach we see in JS tracker. 

I'm the same situation of this guy here (snowplow/snowplow-objc-tracker#409) with a proxy for receiving events. Do you guys have any prediction of when this will work in Android tracker?

Thanks!

### PR DESCRIPTION
**Pull request by theosato**
Fix #319 

Hi, guys. 
I really need a feature like that, once that my application uses a proxy for receiving events. 
I created that issue some time ago but got no answer, so I decided to fork it and do it by myself using the snowplow/snowplow-objc-tracker#409 as an example. 

It's my first time opening a pr in an opensource tool, so please tell me if I'm doing something wrong. :pray: 

Thanks.

### CODE REVIEW

File: snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/Emitter.java [hunk 1]
```diff
@@ -881,6 +898,13 @@ public long getByteLimitPost() {
         return this.byteLimitPost;
     }
 
+    /**
+     * @return the customPath
+     */
+    public String getCustomPath() {
```
Comment (jbeemster) [inline snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/Emitter.java]: This should probably be `getCustomPostPath` to better represent the variable.

**Comment by jbeemster** [comment]
Hi @theosato thanks so much for your PR!  This *should* go into the next release after 1.3.0 but will let @oguzhanunlu confirm...

**Comment by oguzhanunlu** [comment]
Hi @theosato , thanks for the contribution! Please sign the CLA first and then we can incorporate your work into the next release after `1.3.0`.

**Comment by oguzhanunlu** [comment]
@theosato , this is an important issue and we've decided to hold `1.3.0` release to include this. Please sign the CLA at your earliest convenience so that we can proceed.

File: snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/Emitter.java (hunk 1 above)
Comment (theosato) [reply snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/Emitter.java]: Thanks for the tip! It's done. :smile:

**Comment by theosato** [comment]
@oguzhanunlu already signed the forms with CLA stuff. Do I need to do something else in that PR? :slightly_smiling_face:

**Comment by theosato** [comment]
Thanks, man! You rock, guys! :+1:

**Comment by soulrainx** [comment]
Hi @oguzhanunlu, One question here, I was looking in the Snowplow Emitter documentation if there is way to send requests to a custom path on POST for data analysis purposes and I came up with this opened PR.

I see that the Proposed changes were approved but they were not released and they only exist on the release/1.3.0-copy branch , is there any plan to recap them and release or should we fork from here ?

**Comment by AlexBenny** [comment]
Hi @soulrainx , thanks for the heads up. I'll take care to merge this feature in the next Android version.

**Comment by AlexBenny** [comment]
Closing as the code has been imported on PR #356

**Pull request closed without merging**